app/components/meeting-form.js

```diff
@@ -22,11 +23,43 @@ export default class MeetingForm extends Component {
   @service store;
   @service currentSession;
   @service router;
+  @service intl;
+
+  isPublished = trackedFunction(this, async () => {
+    await Promise.resolve();
```

Comment:
Why do we do this? I assume in order to avoid blocking, maybe we want to comment this little hack

Reply:
I also don't know, and I would like to, before merging :)